test_scripts/Defects/5_0/1006/common.lua

```diff
@@ -0,0 +1,52 @@
+---------------------------------------------------------------------------------------------------
+-- Common module for tests of https://github.com/SmartDeviceLink/sdl_core/issues/1006 issue
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local actions = require("user_modules/sequences/actions")
+local hmi_values = require("user_modules/hmi_values")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Module ]]
+local m = { }
+
+--[[ Proxy Functions ]]
+m.Title = runner.Title
+m.Step = runner.Step
+m.preconditions = actions.preconditions
+m.postconditions = actions.postconditions
+m.start = actions.start
+m.registerApp = actions.registerApp
+m.activateApp = actions.activateApp
+m.getMobileSession = actions.getMobileSession
+m.getHMIConnection = actions.getHMIConnection
+m.runAfter = actions.run.runAfter
+m.getHMIAppId = actions.app.getHMIId
+m.policyTableUpdate = actions.policyTableUpdate
+m.getParams = actions.app.getParams
+
+--[[ Common Functions ]]
+function m.getHMIValues()
+  local params = hmi_values.getDefaultHMITable()
+  params.UI.IsReady.params.available = false
+  params.UI.GetCapabilities = nil
+  params.UI.GetLanguage = nil
+  params.UI.GetSupportedLanguages = nil
+  return params
+end
+
+function m.sendRPC(pParams)
+  local cid = m.getMobileSession():SendRPC(pParams.rpc, pParams.requestParam)
+  if pParams.expectExtraRequest then
+    pParams.expectExtraRequest(pParams)
+  end
+  m.getHMIConnection():ExpectRequest("UI." .. pParams.rpc)
+  :Times(0)
+  m.getMobileSession():ExpectResponse(cid, { success = false, resultCode = "UNSUPPORTED_RESOURCE" })
+  m.getMobileSession():ExpectNotification("OnHashChange")
+  :Times(0)
+end
+
```

Comment:
ChangeRegistration, Slider RPCs are not covered.

Reply:
Slider is added
Update issue  https://luxproject.luxoft.com/jira/browse/FORDTCN-13034,  ChangeRegistration is failed